src/backend/db/database_manager.py

```diff
@@ -81,7 +82,43 @@ def add_customer(self):
                 timestamp TIMESTAMP(6) DEFAULT CURRENT_TIMESTAMP(6)
             );
             """
-            session.execute(text(create_table_query))
+            session.execute(text(create_chat_messages_table_query))
+
+            # Creating tickets table if not exists
+            create_tickets_table_query = """
+            CREATE TABLE IF NOT EXISTS tickets (
+                ticket_id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                chat_id VARCHAR(255) NOT NULL,
+                title VARCHAR(255) NOT NULL,
+                description TEXT,
+                priority ENUM('Low', 'Medium', 'High') DEFAULT 'Medium',
+                status VARCHAR(50) DEFAULT 'open',
+                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+                updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+            );
+            """
+            session.execute(text(create_tickets_table_query))
+
+            # Creating custom_fields table if not exists
+            create_custom_fields_table_query = """
+            CREATE TABLE IF NOT EXISTS custom_fields (
+                field_name VARCHAR(255) PRIMARY KEY,
+                field_type ENUM('string', 'integer', 'boolean', 'date') NOT NULL,
```

Comment:
We can add all types supported by database like TEXT, MEDIUM_TEXT etc., Lets add more fields

Why you should name as string, integer

Just provide values like what fields you will give for Databases, so that you dont need a mapper

like field_type ENUM('VARCHAR(255)', 'INT', 'BOOLEAN', 'DATE', 'MEDIUMTEXT') NOT NULL